nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -372,8 +374,20 @@ def get_ignored_names_by_layer_attributes(nncf_graph: NNCFGraph) -> Set[str]:
 
     @staticmethod
     def get_weight_nodes(nncf_graph: NNCFGraph) -> List[NNCFNode]:
-        return [
+        weight_nodes_candidates = [
             node
             for node in nncf_graph.get_all_nodes()
             if issubclass(node.metatype, om.PTOperatorMetatype) and node.metatype.weight_port_ids
         ]
+        weight_nodes = []
+        for node in weight_nodes_candidates:
+            if node.metatype in PTMinMaxAlgoBackend.MAT_MUL_METATYPES and not PTMinMaxAlgoBackend.is_constant_matmul(
```

Comment:
It seems more clear to introduce the private `_is_constant_matmul` method and use it here, instead of the changing current approach in the `MinMaxAlgorithm` class.

Reply:
Also, do we need to keep `get_weight_nodes` as `staticmethod`? Why can't we change and utilize `self.mat_mul_metatypes` here instead of introducing a new `MAT_MUL_METATYPES` variable?

Reply:
1) Please suggest a better name for the `is_constant_matmul` then, how should I name a function, which is checking is matmul a constant or not?
2) You sure everybody will be happy with the introduction of the first non static method for MinMaxBackend?

Reply:
2. Not sure, of course. Let do not change anything, to make almost everyone happy.